src/metabase/lib/metadata/calculation.cljc

```diff
@@ -356,7 +356,9 @@
        {:is-from-previous-stage (= source :source/previous-stage)
         :is-from-join           (= source :source/joins)
         :is-calculated          (= source :source/expressions)
-        :is-implicitly-joinable (= source :source/implicitly-joinable)})
+        :is-implicitly-joinable (= source :source/implicitly-joinable)
+        :is-aggregation         (= source :source/aggregations)
```

Comment:
@bshepherdson I'll need your blessing for these added props

Reply:
LGTM